docs/user/rstudio/_quarto.yml

```diff
@@ -3,44 +3,52 @@ project:
 
 website:
   title: "RStudio User Guide"
-  favicon: images/RStudio-ball.svg
   twitter-card: true
   open-graph: true
+  favicon: images/favicon.svg
   page-navigation: true
   site-url: https://docs.rstudio.com/ide/user/
-  repo-url: https://github.com/rstudio/rstudio/tree/main/docs/user/rstudio
+  repo-url: https://github.com/rstudio/rstudio/
+  repo-subdir: "docs/user/rstudio/"
   repo-actions: [edit, issue]
   page-footer:
     left: |
-      Copyright © 2022 Posit Software, PBC. All Rights Reserved.
+      Copyright © 2022 Posit, PBC. All Rights Reserved.
     right:
       - icon: house-fill
-        href: https://www.rstudio.com
+        href: https://www.posit.co
       - icon: question-circle-fill
         href: https://support.rstudio.com/hc/en-us
       - icon: lightbulb-fill
         href: https://solutions.rstudio.com
       - icon: twitter
         href: https://twitter.com/rstudio
   navbar:
+    title: "RStudio User Guide<br>{{< var builtType >}} {{< var version >}}"
+    collapse: false
     pinned: true
     search: true
     logo: images/posit-logo-fullcolor-TM.svg
     left:
-      - href: index.qmd
-        text: "Overview"
+      # - href: index.qmd
+      #   text: "Overview"
       - href: ide/get-started/index.qmd
         text: "Getting Started"
       - href: ide/guide/ui/ui-panes.qmd
         text: "Guide"
       - href: ide/reference/shortcuts.qmd
         text: "Reference"
     right:
-      - href: "https://posit.co"
-        text: posit.co
-      - href: "https://www.rstudio.com/products/rstudio/release-notes/"
-        text: "Release Notes"
-
+      - text: "Release Notes"
+        url: "https://www.rstudio.com/products/rstudio/release-notes/"      
+      - icon: house
+        url: https://docs.rstudio.com/
+        aria-label: Posit Documentation website
+      - icon: question-circle
+        url: https://support.rstudio.com/
+        aria-label: Posit Professional Support
+      - text: "![](https://user-images.githubusercontent.com/29187501/195717097-e8dda415-8d24-4011-8367-18ba9c83eb5b.png)"
```

Comment:
Hey @jthomasmock - Sam provided a different "fix" for the styles that seem to be more appropriate than my solution.

`quarto.yaml`:
```
- text: "<img src='/images/posit-logo-black-TM.png' id='navbar-right-logo'>"
        url: "https://posit.co/"
```

`scss`:
```
#navbar-right-logo {
  width: 70px;
  min-width: 70px;
}
```

Reply:
Noice - thanks!